### DISCUSSION TITLE
Submit success and errors messages

### DISCUSSION THREAD

**Discussion by rrg92** [formio/formio.js#5072] [General]
I have this form:

```js
var formSchema = MyForm.schema // MyForm is valid json created with builder
let renderedForm = await Formio.createForm(document.getElementById('MyForm'), formSchema);

renderedForm.on('submit', function(sentData,saved){ 
	console.log('user sent', sentData, saved);  
	renderedForm.emit('submitDone', {}) 
	SetStatus('Check console to rendered data...');
})			
```

When the user submits data, the message "Submission completed" with a green background doesn't stay and the button changes to the success layout and quickly goes back...

The same thing happens when I emit the submitError event.

I want the form to show a success message for the user after they submit their message, and allow them to close it. Alternatively, if there's an error, all results from my API should be shown to the user in the alert dialog.

How can I achieve this?

Thanks for any help!

**Comment by edwinanciani** [comment]
Could you try to emit the event with no arguments?  renderedForm.emit('submitDone')
